tests/unit/amo/components/TestSearchResult.js

```diff
@@ -92,9 +92,10 @@ describe(__filename, () => {
   });
 
   it('links the heading to the detail page with a source', () => {
+    const _config = getFakeConfig({ enableFeatureUseUtmParams: false });
```

Comment:
If the plan is to turn the feature on, wouldn't it make more sense for the test to assume the feature is on, and not off? Are you just doing this to keep the patch small, and deferring updating the test to a different patch?